localization/english/dynasties/wc_dynasty_names_l_english.yml

```diff
@@ -1645,7 +1645,7 @@
  dynn_Ghgmlmr:0 "Ghgmlmr"
  dynn_Ghhgmh:0 "Ghhgmh"
  dynn_Ghhmlg:0 "Ghhmlg"
- dynn_Ghlml:0 "Ghlml"
+ dynn_Grimscale:0 "Grimscale"
```

Comment:
I don't think removing the original is necessary
They could be used as a random name for their namelist

Reply:
i agree. more is better. please readd them @Malicos

Reply:
readded